gapic/ads-templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2

```diff
@@ -1048,15 +1048,23 @@ def test_{{ method.name|snake_case }}_rest(request_type):
         # Wrap the value into a proper Response obj
         response_value = Response()
         response_value.status_code = 200
-    {% if method.void %}
+        {% if method.void %}
         json_return_value = ''
         {% elif method.lro %}
         json_return_value = json_format.MessageToJson(return_value)
-    {% elif method.server_streaming %}
-        json_return_value = "[{}]".format({{ method.output.ident }}.to_json(return_value))
-    {% else %}
-        json_return_value = {{ method.output.ident }}.to_json(return_value)
-    {% endif %}
+        {% else %}
+
+        {% if method.output.ident.is_proto_plus_type %}
+        pb_return_value = {{ method.output.ident }}.pb(return_value)
+        {% else %}
+        pb_return_value = return_value
```

Comment:
(minor): This is an unnecessary assignment., and IIUC we don't use `return_value` after this point. So could we s/`pb_return_value`/`return_value`/ in lines 1058 and 1062? Then we don't need this `else` clause

(There's also another block similar to this one below in this file, and the same comment applies)

Reply:
Fixed in 350006c